<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportSettingsDialog</class>
 <widget class="QDialog" name="ExportSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Video</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="cameraTab">
      <attribute name="title">
       <string>Single Camera</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>16</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <property name="spacing">
        <number>8</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="cameraPreviewWidget" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="nx::client::desktop::ui::ExportSettingsWidget" name="cameraExportSettingsPage"/>
         <widget class="nx::client::desktop::ui::TimestampOverlaySettingsWidget" name="timestampSettingsPage"/>
         <widget class="nx::client::desktop::ui::ImageOverlaySettingsWidget" name="imageSettingsPage"/>
         <widget class="nx::client::desktop::ui::TextOverlaySettingsWidget" name="textSettingsPage"/>
         <widget class="nx::client::desktop::ui::RapidReviewSettingsWidget" name="rapidReviewSettingsPage"/>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QWidget" name="buttonsWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>16</number>
          </property>
          <item>
           <widget class="nx::client::desktop::ui::SelectableTextButton" name="cameraExportSettingsButton">
            <property name="text">
             <string>Export Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="overlayButtonsWidget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="nx::client::desktop::ui::SelectableTextButton" name="timestampButton">
               <property name="text">
                <string notr="true">&lt;Timestamp&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="nx::client::desktop::ui::SelectableTextButton" name="imageButton">
               <property name="text">
                <string notr="true">&lt;Image&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="nx::client::desktop::ui::SelectableTextButton" name="textButton">
               <property name="text">
                <string notr="true">&lt;Text&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="nx::client::desktop::ui::SelectableTextButton" name="speedButton">
               <property name="text">
                <string notr="true">&lt;Speed&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutTab">
      <attribute name="title">
       <string>Multi Video</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>16</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <property name="spacing">
        <number>8</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="layoutPreviewWidget" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="QStackedWidget" name="layoutExportSettingsWidget">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="layoutExportSettingsPage"/>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QWidget" name="layoutButtonsWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>16</number>
          </property>
          <item>
           <widget class="nx::client::desktop::ui::SelectableTextButton" name="layoutExportSettingsButton">
            <property name="text">
             <string>Export Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QnPanel" name="filePathPanel" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <property name="spacing">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="folderLabel">
        <property name="text">
         <string>Folder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="folderLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fileNameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="browseFolderButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="fileNameLineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="formatComboBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::client::desktop::ui::ExportSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/client/desktop/export/ui/widgets/export_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::client::desktop::ui::SelectableTextButton</class>
   <extends>QPushButton</extends>
   <header location="global">nx/client/desktop/ui/common/selectable_text_button.h</header>
  </customwidget>
  <customwidget>
   <class>nx::client::desktop::ui::TimestampOverlaySettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/client/desktop/export/ui/widgets/timestamp_overlay_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::client::desktop::ui::ImageOverlaySettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/client/desktop/export/ui/widgets/image_overlay_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::client::desktop::ui::TextOverlaySettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/client/desktop/export/ui/widgets/text_overlay_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::client::desktop::ui::RapidReviewSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/client/desktop/export/ui/widgets/rapid_review_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnPanel</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/common/panel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
